Doc 9222

tonyjhillman opened the pull request.

tonyjhillman committed:
DOC-9222: Draft release notes for 7.0.2

tonyjhillman committed:
DOC-9222: Release Notes for 7.0.2

tonyjhillman committed:
DOC-9222: Additional fixed issues for release notes for 7.0.2

tonyjhillman committed:
DOC-9222: Fixed links

tonyjhillman committed:
DOC-9222: Added to new features section for release notes

tonyjhillman committed:
DOC-9222: Additional fixes and modifications

b33f commented on modules/release-notes/pages/relnotes.adoc:
```diff
@@ -1,10 +1,306 @@
+| *Summary*: Add version check to enable plasma indexes on Ephemeral buckets from 7.0.2
+
+| https://issues.couchbase.com/browse/MB-48145[MB-48145^]
+| *Summary*: [Enforce-TLS] IndexStatus returns Cannot communicate with indexer process. Information on indexes may be stale. Will retry
```
Should remove this one, Enforce TLS didn't exist before this release.

tonyjhillman replied:
Removed.

b33f commented on modules/release-notes/pages/relnotes.adoc:
```diff
@@ -1,10 +1,306 @@
+| *Summary*: CPU utilization is much higher in 7.0.1/2 when cluster is idle
+
+| https://issues.couchbase.com/browse/MB-48030[MB-48030^]
+| *Summary*: [TLS] Strict mode, see errors in query log for 9101 port for indexer client
```
Should remove this one, TLS strict mode didn't exist before this release.

tonyjhillman replied:
Removed.

b33f commented on modules/release-notes/pages/relnotes.adoc:
```diff
@@ -1,10 +1,306 @@
+This section highlights the notable new features and improvements in this release.
+
+* TLS can be specified as mandatory for all internal and external cluster-communications &#8212; see xref:manage:manage-security/manage-tls.adoc[Manage On-the-Wire Security].
+
```
We also added a new feature to allow administrators to enable HSTS (HTTP Strict Transport Security)

tonyjhillman replied:
I have added "* HSTS (HTTP Strict Transport Security) can now be enabled."

b33f commented on modules/release-notes/pages/relnotes.adoc:
```diff
@@ -1,10 +1,306 @@
+| *Summary*: [Enforce-TLS] IndexStatus returns Cannot communicate with indexer process. Information on indexes may be stale. Will retry
+
+| https://issues.couchbase.com/browse/MB-48146[MB-48146^]
+| *Summary*: [IPV6] no index available when switching to ipv6 or ipv6only IP family
```
Remove this one, ipv6only didn't exist before this release.

tonyjhillman replied:
Removed.

b33f commented on modules/release-notes/pages/relnotes.adoc:
```diff
@@ -1,10 +1,306 @@
+| *Summary*: XDCR - collect remote clusters and replication info as part of cbcollect
+
+| https://issues.couchbase.com/browse/MB-47546[MB-47546^]
+| *Summary*: Bind only to IPv4 addresses when invoked with IPv4-Only cluster-wide setting - xdcr
```
Remove this one, IPV4-Only didn't exist before this release.

tonyjhillman replied:
Removed.

b33f commented on modules/release-notes/pages/relnotes.adoc:
```diff
@@ -1,10 +1,306 @@
+| *Summary*: Bind only to IPv4 addresses when invoked with IPv4-Only cluster-wide setting - xdcr
+
+| https://issues.couchbase.com/browse/MB-47723[MB-47723^]
+| *Summary*: Enforce TLS - XDCR
```
Remove this one, it's a new feature, not an issue

tonyjhillman replied:
Removed.

tonyjhillman committed:
DOC-9222: Incorporated review feedback

jon-strabala commented:
Hi Tony there is also an update for Eventing concerning some new functionality.

The Eventing service now supports a bucket backed caching capability to improve performance for repetitive Data Service (KV) lookups.

tonyjhillman commented:
I have added:
* The Eventing Service now supports a bucket-backed caching capability, to improve performance for repetitive Data Service lookups.

tonyjhillman committed:
DOC-9222: Added line on Eventing feature

tonyjhillman committed:
DOC-9222: Added MB-48713

mihirkamdarqe commented:
**Following is the feedback for the Index service section**

https://issues.couchbase.com/browse/MB-47833 - this is a regression introduced in 7.0.2 itself, so I dont think its necessary to release note it. 
https://issues.couchbase.com/browse/MB-48144 - this is a regression introduced in 7.0.2 itself, so I dont think its necessary to release note it. 
https://issues.couchbase.com/browse/MB-48366 - this is a regression introduced in 7.0.2 itself, so I dont think its necessary to release note it. 
https://issues.couchbase.com/browse/MB-47737 - there is no functional impact. I dont see the need to release note this
https://issues.couchbase.com/browse/MB-48336 - Summary can be reworded to "Rollback stats will now be persisted in case of indexer crash" or better
https://issues.couchbase.com/browse/MB-48017 -  - this is a regression introduced in 7.0.2 itself, so I dont think its necessary to release note it. 
https://issues.couchbase.com/browse/MB-47878 - Reword it - "Increase polling interval in pollForDeletedBuckets in projector" or better
https://issues.couchbase.com/browse/MB-47760 - Reword it

tonyjhillman commented:
For the Index Service, I have removed MBs 47833, 48144, 48366, 47737, and 48017, as specified. 

For 48366 and 47878, I have added the specified rewording.

For 47760, I have reworded as follows: Panic caused by extraneous unlocking.
Let me know if an alternative rewording for 47760 is required.

tonyjhillman committed:
DOC-9222: Added feedback on Index Service tickets

arod1987 commented:
Looks okay for XDCR and Tools

jon-strabala commented:
Looks okay for Eventing

RayOffiah reviewed (approved):
As far as I can tell, looks okay.

mihirkamdarqe commented:
Should we document https://issues.couchbase.com/browse/MB-48783 ?
